src/routes/gallery/+page.svelte

```diff
@@ -1,11 +1,13 @@
 <script lang="ts">
-  import { goto } from '$app/navigation'
   import { onDestroy } from 'svelte'
+  import { goto } from '$app/navigation'
 
-  import { galleryStore, sessionStore, themeStore } from '../../stores'
-  import { AREAS } from '$lib/gallery'
-  import Dropzone from '$components/gallery/upload/Dropzone.svelte'
-  import ImageGallery from '$components/gallery/imageGallery/ImageGallery.svelte'
+  import { sessionStore } from '../../stores'
+  import { themeStore } from '../../stores'
```

Comment:
Could we consolidate `sessionStore` and `themeStore` into a one import statement?